eQuiz/modules/eQuiz.Web/Areas/Admin/Scripts/Templates/Review.html

```diff
@@ -0,0 +1,61 @@
+﻿<div ng-controller="ReviewController as rc">
```

Comment:
1. Do not use the "Templates" folder, put the file into the root of the Scripts folder.
2. Use lower case for file's name - review.html